server/app/services/program/predicate/LeafAddressServiceAreaExpressionNode.java

```diff
@@ -0,0 +1,78 @@
+package services.program.predicate;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import java.util.Optional;
+import services.question.types.QuestionDefinition;
+
+/** Represents an assertion that an address question is within a given service area. */
+@JsonTypeName("leafAddressServiceArea")
+@AutoValue
+public abstract class LeafAddressServiceAreaExpressionNode
+    implements ConcretePredicateExpressionNode {
+
+  @JsonCreator
+  public static LeafAddressServiceAreaExpressionNode create(
+      @JsonProperty("questionId") long questionId,
+      @JsonProperty("serviceAreaLabel") String serviceAreaLabel) {
```

Comment:
I'm a little concerned about using the service area label instead of the value. The service area value isn't going to change, but what if the label is changed after an address predicate is added to a program?

Reply:
Renamed to `serviceAreaId`